docs/src/app/components/pages/components/BottomNavigation/Page.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Title from 'react-title-component';
+
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import bottomNavigationReadmeText from './README';
+import BottomNavigationExampleSimple from './ExampleSimple';
+import bottomNavigationExampleSimpleCode from '!raw!./ExampleSimple';
+import bottomNavigationCode from '!raw!material-ui/BottomNavigation/BottomNavigation';
+import bottomNavigationItemCode from '!raw!material-ui/BottomNavigation/BottomNavigationItem';
+
+const descriptions = {
```

Comment:
It would be better to put the description about the class example definition as a comment. React Docgen will picked it up.